src/System.Security.Cryptography.OpenSsl/src/project.json

```diff
@@ -9,8 +9,8 @@
     "System.Runtime": "4.0.20",
     "System.Runtime.InteropServices": "4.0.20",
     "System.Runtime.Numerics": "4.0.0",
-    "System.Security.Cryptography.Algorithms": "4.1.0-rc3-24102-00",
     "System.Security.Cryptography.Encoding": "4.0.0-rc3-24102-00",
+    "System.Security.Cryptography.Primitives": "4.0.0-rc3-24102-00",
```

Comment:
I'm surprised that the reference to primitives is required, since it's in the closure from Encoding...

This file feels like it should be revertable, since we shouldn't P2P Algorithms.